services/QuillLMS/app/controllers/cms/school_admins_controller.rb

```diff
@@ -54,18 +60,19 @@ def set_school
 
   private def create_admin_user_for_existing_user(user, new_user: false)
     school_id = params[:school_id]
+    school_admin_for_school = SchoolsAdmins.find_by(school_id: school_id, user_id: user.id)
```

Comment:
Since the school name is what we're really after here, what about going straight to that:

```
school_name = SchoolsAdmins.find_by(school_id: school_id, user_id: user.id)&.school&.name

if school_name
  return render json: { message: t('admin_created_account.existing_account.admin.linked', school_name: school_name) }
end
```